src/third-party/PhishingController.ts

```diff
@@ -135,18 +137,26 @@ export class PhishingController extends BaseController<
       return;
     }
 
-    const phishingOpts = await this.queryConfig(this.configUrl);
+    const [phishingOpts, pfBlacklist] = await Promise.all([
+      await this.queryConfig<EthPhishingResponse>(this.configUrlMetaMask),
+      await this.queryConfig<string[]>(this.configUrlPhishFortHotlist),
+    ]);
+
     if (phishingOpts) {
+      if (pfBlacklist) {
+        phishingOpts.blacklist.push(...pfBlacklist);
```

Comment:
What if we update the `EthPhishingResponse` interface to contain an optional pfBlacklist []? 

This would not break current implementation and provide a way to identify what list the url originated. The only issue I see with this implementation is that it is not very scalable but I don't know how many other lists we may want to integrate. WDYT? @Gudahtt

Reply:
> 

I like the idea Seth, will wait to hear @Gudahtt's input before making any changes to the PR on our side.

Reply:
My only other thought to make it a bit more scalable is to use a `externalBlacklists: Dictionary<T>?`. Where the key is the report URL and the value is the []. This way we can append as many blacklists as we want.

Reply:
I second this comment

Reply:
I second this comment

Reply:
>This would not break current implementation and provide a way to identify what list the url originated.

Agreed that we need to know where the entry originated. When we tell users that a site is blocked, we provide guidance on how to report incorrectly blocked sites. We need to know where to tell them to go in that situation, we don't want people spamming our repo asking us to unblock sites that are only blocked by phishfort.

One issue with just storing it as a separate list is that it might increase the memory usage of this controller a fair bit. Perhaps we could exclude any entries that are on our main blocklist already, to minimize memory usage.

Reply:
Making this scalable to allow other external blocklists sounds like a fine idea to me, but I don't think it needs to be a blocker IMO. We can get this merged first, this one additional list is the more important one. Adding other lists needs a bunch more UI work to be useful in practice, and that hasn't been planned or designed yet, so we have time.

Reply:
@moosthuizen42 @Gudahtt   What do you think about converting the `phishingOpts.blacklist` to a `phishingOpts.blocklist Dictionary<blockedUrl:urlToReportIncorrectlyBlockedSite>`? 

You would need to add a resolution so duplicates are not to pushed into the dictionary. This dictionary would only contain the keys being a unique set of blockedUrls and values being the corresponding incorrectly blocked report url. If there is a duplicate found between the two lists use the `phishingOpts` report url (MM Report URL: https://github.com/metamask/eth-phishing-detect).

Reply:
That might result in even more memory usage than before because of the duplication of the reporting URL.

Or maybe it would get optimized away, who knows. In retrospect, my concern about memory usage was probably premature and I would not block on resolving that. We can measure perf and optimize later.

The critical part is that the lists are separate so we know where to direct users.

Reply:
Resolved by the PR to `eth-phishing-detect` by @Gudahtt.